mfr/settings.py

```diff
@@ -84,6 +84,9 @@ def child(self, key):
 UNSUPPORTED_EXPORTER_MSG = 'Exporting of this file type is not currently supported.'
 UNSUPPORTED_RENDER_MSG = 'Viewing of this file type is not currently supported. Please download the file to view.'
 
+
+MAX_FILE_SIZE_TO_RENDER = {'csv': 100000, '.xlsx': 100000, '.tsv': 100000, '.sav': 100000, '.xls': 100000}
```

Comment:
I think `csv` should be `.csv`.  Also, I think the ticket said to start with 100Mb, but this is 100Kb, yes?